### ISSUE TITLE
Blank pages using http server mode and text archive.org DBs

### ISSUE THREAD

**Issue by thombles** [gnosygnu/xowa#60]
Hi again, I have the bad news is that v3.6.3.3 continues to render blank pages for me. I have deleted everything except the `wiki/` directory and re-extracted the zip file, leaving everything at defaults. I assume XOWA does not modify the wiki databases so they should be the same.

I run it like this:

    $ java -jar xowa_linux_64.jar --app_mode http_server --http_server_port 8100

I search for the word "cog" and the search results display great. :) I then click on the top result "Cog". I see this in the server console:

    10.130.1.1:8100|GET|/en.wikipedia.org/wiki/Cog
    10.130.1.1:8100|GET|/fsys/bin/any/xowa/html/res/src/xowa/search-suggest/search-suggest.js
    10.130.1.1:8100|GET|/fsys/user/anonymous/wiki/en.wikipedia.org/html/xowa_common.css
    10.130.1.1:8100|GET|/fsys/user/anonymous/wiki/en.wikipedia.org/html/xowa_wiki.css
    10.130.1.1:8100|GET|/fsys/bin/any/xowa/html/res/src/xowa/core/core.css
    10.130.1.1:8100|GET|/fsys/bin/any/xowa/html/res/src/xowa/popups/popups.css
    10.130.1.1:8100|GET|/fsys/bin/any/xowa/html/res/src/xowa/search-suggest/search-suggest.css
    10.130.1.1:8100|GET|/fsys/bin/any/xowa/html/res/src/xowa/core/core.js
    10.130.1.1:8100|GET|/fsys/bin/any/xowa/html/res/src/xowa/core/exec.js
    10.130.1.1:8100|GET|/fsys/bin/any/xowa/html/res/src/xowa/core/DOMContentLoaded.js
    10.130.1.1:8100|GET|/fsys/bin/any/xowa/html/res/src/xowa/find/xowa_find_html.css
    10.130.1.1:8100|GET|/fsys/bin/any/xowa/html/res/src/xowa/find/xowa_find_html.js
    10.130.1.1:8100|GET|/fsys/bin/any/xowa/html/res/lib/jquery/jquery-1.11.3.min.js
    10.130.1.1:8100|GET|/fsys/bin/any/xowa/html/res/src/xowa/popups/popups.js
    10.130.1.1:8100|GET|/fsys/bin/any/xowa/file/app.general/twisty_right.png
    10.130.1.1:8100|GET|/fsys/bin/any/xowa/html/res/src/gadgets/navframe/mediawiki.gadget.navframe.js
    10.130.1.1:8100|GET|/fsys/user/anonymous/wiki/en.wikipedia.org/html/logo.png
    10.130.1.1:8100|GET|/fsys/bin/any/xowa/file/app.window/app_icon.png

The page renders like this:

![xowa-cog-render](https://cloud.githubusercontent.com/assets/5511175/16260037/c296dbfc-38a8-11e6-883f-fa70dba1d11e.png)

The relevant HTML section is this:

    <div id="mw-page-base" class="noprint"></div>
    <div id="mw-head-base" class="noprint"></div>
    <div id="content" class="mw-body">
    <div id='xowa_err_div' class='xowa_err' style='display:none; color:red; white-space:pre; display:block; unicode-bidi:embed;'></div><h1 id="firstHeading" class="firstHeading"><span>Cog</span></h1>  
    <div id="bodyContent" class="mw-body-content">
        <div id="siteSub">From XOWA: the free, open-source, offline wiki application</div>
        <div id="contentSub" class="visible"></div>
        <div id="jump-to-nav" class="mw-jump">Jump to:<a href="#mw-navigation">navigation</a>, <a href="#p-search">search</a></div>
        <div id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr">
        <div class="visualClear">
    <br/><br/>
            This page was last modified on 2016-02-20, at 04:36.
            <div style='float:right'>
            <ul style='-moz-column-count: 2; list-style:none;font-size:xx-small;'>
                <li>version</li>
                <li>build time</li>
                <li>3.6.3.3</li>
                <li>2016-06-21 05:48</li>
            </ul>
            </div>
            <br/>
        </div>
        </div>
    </div>
    </div>

Let me know if there's anything else that would be useful!

**Comment by gnosygnu** [comment]
Hi! Thanks for the screenshots and the excellent detail.

This was an oversight on my part. v3.6.3.3 fixed the issue for the GUI, but not the HTTP Server. I didn't test the HTTP Server, because I assumed they were using the same code. I forgot that I still hadn't merged the code so that both GUI and HTTP Server did the same thing.

I merged the code for v3.6.3.4 and released it now. See: https://github.com/gnosygnu/xowa/releases/tag/v3.6.3.4 . I also tested with both GUI and HTTP Server now with the version of English Wikipedia from archive.org.

Let me know if there's anything else. Thanks again!
